spec/features/stories_spec.rb

```diff
@@ -39,6 +39,49 @@
       end
     end
 
+    context 'when the user save a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'highlights story on click to save', js: true do
+        find(story_selector(story)).trigger('click')
+        expect(page).to have_css('.editing')
```

Comment:
Can we have a `expect(story_selector(story)).to have_css('.editing')` here? Having this class somewhere on this page doesn't guarantee that it's in the story we clicked 😄

Reply:
Also, does it blink when the save fails? Is it possible to add a test for this case?